CONTRIBUTING.md

```diff
@@ -2,68 +2,68 @@
 
-## API
-<div id="api">
+## API conventions
 
-This section heavily relies on the definitions from our [terminology guide](TERMINOLOGY.md). Specifically, we use the terms "reduction", "sensitive features", "moment", and "parity" in the following.
+This section relies on the definitions from our [terminology guide](TERMINOLOGY.md), including the definitions of "estimator", "reduction", "sensitive features", "moment", and "parity".
 
-For all disparity mitigation methods algorithm-specific parameters are passed to the constructor. The methods to fit a mitigator and predict values with the resulting model shall resemble the APIs used by scikit-learn as much as possible for the sake of ease of use. Any deviations are noted below.
+### Unfairness mitigation algorithms
 
-### Reductions
+Unfairness mitigation algorithms take form of scikit-learn-style estimators. Any algorithm-specific parameters are passed to the constructor. The resulting instance of the algorithm should support methods `fit` and `predict` with APIs resembling those of scikit-learn as much as possible. Any deviations are noted below.
 
-Reductions require an estimator to be passed that implements the `fit` method with the `sample_weight` argument. The constraints for reductions are all moments (see `fairlearn.reductions`) passed as instances of classes inheriting from `Moment`. Moments are vector functions that we use to formalize our constraints. The moment objects need to be passed as constraints to the constructor of a reduction technique, which internally uses the constraints to solve the optimization problem.
+#### Reductions
+
+Reduction constructors require a parameter corresponding to an estimator that implements the `fit` method with the `sample_weight` argument. Parity constraints for reductions are expressed via instances of various subclasses of the class `fairlearn.reductions.Moment`. Formally, instances of the class `Moment` implement vector-valued random variables whose sample averages over the data are required to be bounded (above and/or below).
 
 ```python
 constraints = Moment()
-reduction = Reduction(estimator, objective=objective, constraints=constraints, **kwargs)
+reduction = Reduction(estimator, constraints)
 ```
 
-Reduction-based disparity mitigation algorithms (such as the ones under `fairlearn.reductions`) provide `fit`, `predict`, and `_pmf_predict` methods with the following signatures:
+Reductions  provide `fit`, `predict`, and `_pmf_predict` methods with the following signatures:
```

Comment:
Do we want to publicise `_pmf_predict` or should we keep that private for now?

Reply:
good point. removing until we standardize.